template.html

```diff
@@ -1,50 +1,83 @@
-<!doctype html>
```

Comment:
There is no need for the doctype to be capitalised, I think lowercase is more appropriate as other HTML tags are also.